launchpod/src/main/webapp/angular-launchpod/src/app/app.module.ts

```diff
@@ -14,12 +14,15 @@ import { MatButtonModule } from '@angular/material/button';
 import { MatSidenavModule } from '@angular/material/sidenav';
 import { MatIconModule } from '@angular/material/icon';
 import { MatListModule } from '@angular/material/list';
+import { ClipboardModule } from '@angular/cdk/clipboard';
```

Comment:
same to these imports, since it is alphabetized you should move ClipboardModule to the top

Reply:
done.